packages/ag-charts-community/src/dom/elements.ts

```diff
@@ -87,6 +97,6 @@ function applyAttrs(element: HTMLElement, attrs?: Attrs) {
 }
 
 function normaliseMissingAriaAttrs(key: string) {
-    if (key === 'ariaControls') return 'aria-controls';
-    return key;
+    // Convert camelCaseString to kebab-case-string
+    return key.replace(/([a-z])([A-Z])/g, '$1-$2').toLowerCase();
```

Comment:
This won't work for some property-attribute pairs like `ariaSetSize` / `aria-setsize` or `ariaPosInSet` / `aria-posinset`. We might want to have a more strict interface for the `Attrs` type.

Thoughts?

Reply:
Agreed, the current `Attrs` type also allows function properties like `setEventListener` which is a bit leaky.

If we only care about targetting aria attributes, could we just replace the `aria` with `aria-` and not kebab the whole thing? Or are there other attrs we need to consider?

The reason I called it "missing" aria attrs is the typescript types for elements includes many attrs like `ariaLabel` but not `ariaControls`. Not sure if that is just our ts version being out of date or if it implies something else?